biometric-login/test/configs/wdio.android.sauce.emu.conf.js

```diff
@@ -96,7 +117,16 @@ config.capabilities = [
         locale: 'en',
         // Group builds by build name
         build: buildName,
+        name: 'Android 7.1 Biometric Login',
     },
 ];
 
+const appiumVersion = process.env.npm_config_appium_version
```

Comment:
Can you put the constant at the top of the file, it's more of a preferred way instead of a mandatory one

Reply:
moved into a shared conf